_data/accessibility-tests/header.yml

```
@@ -0,0 +1,243 @@
+title: Header
+component_status: pass
+test_items:
+# General tests
+  - summary:
+    summary_additional:
+    test_status:
+    test_type: general
+    version_tested:
+    wcag_criterion:
+  - summary: Header content is consistent in mobile views.
+    summary_additional: |
+      When you view the page on a mobile device, 
+      the header is visible and works in both portrait and landscape settings.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.4
+  - summary: Text in header meets color contrast requirements.
+    summary_additional: |
+      When you use a color contrast checker to view the header,
+      contrast between the text color and background color is at least 4.5:1.
+    test_status: 3.8.1
+    test_type: general
+    version_tested: pass
+    wcag_criterion: 1.4.3
+  - summary: Header remains visible and functional when screen is magnified. 
+    summary_additional: |
+      When you zoom in to 400%, 
+      you can still see and use the header menu and navigation, 
+      and it doesn't overlap with or get cut off by any other content.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+  - summary: Search icon in header meets color contrast requirements.
+    summary_additional: |
+      When you use a color contrast checker to view the search icon, 
+      contrast between the icon and adjacent colors is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.11
+  - summary: Current page link has a distinct style.
+    summary_additional: |
+      When viewing the navigation link list, 
+      the current page has distinct styling that gives clear visual indication of where you are in the site.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.8
+  - summary: Header appears consistently.
+    summary_additional: |
+      The header appears in the same location on every page of your government website.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.3
+  - summary: All clickable elements touch target size meet height and width requirements.
```

@finekatie 

**question (non-blocking):** Wondering if we can re-phrase this a little bit for clarity.

```suggestion
  - summary: The touch target on all interactive elements meets height and width requirements.
```

---

Resolved in [f0fd250](https://github.com/uswds/uswds-site/pull/2761/commits/f0fd250b15c12b1f60573977565ce6b5a13bd715)